neomodel/relationship_manager.py

```diff
@@ -121,6 +121,36 @@ def connect(self, node, properties=None):
 
         return rel_instance
 
+    @check_source
+    def connect(self, node, properties=None):
+        """
+        Connect a node
+
+        :param node: a node instance which will be connected to the source node
+        :param properties: for the new relationship
+        :type: dict
+        :return:
+        """
+        self._check_node(node)
+
+        q = "MATCH (them), (us) WHERE id(them)=$them and id(us)=$self " \
+            "MERGE"
+        return self.connect_helper(q, properties, node)
+
+    @check_source
+    def bulk_connect(self, nodes: List[UUID], properties=None):
+        """
+        Connect a list of nodes to the source node.
+
+        Unlike `connect()`, this function receives a list of node UUIDs,
+        not a node instance. Using UNWIND, the query connects all nodes in
+        the list to the source node.
```

Comment:
I would add the docstring annotatoin for the params, similar to how it is in `connect`, so we maintain code quality / documentation vis a vis the upstream, in case we ever want to attempt to merge it back.